tests/helpers/TestAdminSecretChecker.py

```diff
@@ -38,7 +38,7 @@ def test_createResource(self, b64encode_mock):
         result = self.checker.createResource(self.cluster_object)
         self.assertEqual(self.kubernetes_service.createSecret.return_value, result)
         self.kubernetes_service.createSecret.assert_called_once_with(
-            self.secret_name, "default", {"username": "root", "password": "random-password"}
+            self.secret_name, self.cluster_object.metadata.namespace, {"username": "root", "password": "random-password"}
```

Comment:
In tests it's better to use 'hardcoded' values for things like this, otherwise we fail to catch changes because the code/config change automatically propagates in the tests.